actioncable/lib/action_cable/server/configuration.rb

```diff
@@ -15,13 +15,15 @@ def initialize
         @connection_class  = ApplicationCable::Connection
         @worker_pool_size  = 100
 
-        @channels_path = Rails.root.join('app/channels')
+        @channels_path = [Rails.root.join('app/channels')]
 
         @disable_request_forgery_protection = false
       end
 
       def channel_paths
-        @channels ||= Dir["#{channels_path}/**/*_channel.rb"]
+        @channels ||= channels_path.collect do |channel_path|
+          Dir["#{channel_path}/**/*_channel.rb"]
+        end.flatten
```

Comment:
I would suggest `channel_load_paths` because it would be harder to get confused.